### ISSUE TITLE
Runeblade of baron rivendare health regen scales with SP

### ISSUE THREAD

**Issue by rhoenluciusellouan** [LightsHope/issues#925]
On the previous server Baron rivendares runeblades effect never worked properly. It should give the movement speed increase, 20 health per 5 and an additional health per 5 for every point of spell power. This is how the weapon has worked from launch till present day

https://www.wowhead.com/item=13505/runeblade-of-baron-rivendare#comments

ctrl+f search zansky, his post shows the effect.

the effect was still apparently active as of 7.0

**Comment by theloras** [comment]
yep - I never thought to add this weapon to the weapons/items in the bug report for Drain Life (ie: Corrupted Ashbringer et al)


_By Zansky on 05/30/2010 (Patch 3.1.3)
The life regen upon equiping seems to be 30hp5 increased by ~100% of your spellpower
On my ret pala it's ticking well over 1.5k hp every 5 seconds, which makes it awesome for passive healing, should you ever need that_

_By harmil on 02/08/2011 (Patch 3.1.3)
Has this item been massively buffed of late?_

_I got this in an LFD group on my pally. I never really used it because I'm prot almost all the time, but I wanted to catch up to a friend's leveling toon, so I went and did some questing. After a while I noticed my health was never dipping below 90%. I thought that was odd, but let it go. Tonight I took some heavy damage in the mines in Blasted Lands and *bang* I was back to full health in a few seconds as I turned in the quest. I'm a bit out of touch with Ret mechanics, so I thought there was some passive self-healing that was going on, but that seemed a bit unbalanced..._

_So tonight, I checked and indeed, it's the sword. Since it has no STA, it was easy to test. I put on my highest STA gearset (pre-TBC, mind you) which got me up to 4131 health. Naked, I'm at 2361. Taking off all of that gear and putting it on with the sword equipped and without it, I was able to generate the following numbers:_

_With sword: 1770/26sec = 68.1 health/sec
Without sword (empty slot): 1770/84sec = 21.1 health/sec_

_So, the sword is giving me around 50 health per second (my stopwatch skills notwithstanding). That's quite a lot more than the 20 per 5 that was quoted above! It's going to be hard to give this thing up in Outland... sigh. But I have to wonder. What does that scale with? Anyone have one of these at max level that can test to see if it scales at all, or do you see around 50h/s?_
http://classicdb.ch/?item=13505#comments

_Zeliek
<Alliance Traitors>
120 Human Paladin
4700
322 posts
1 day ago
Runeblade of Baron Rivendare's unholy aura scaled with spell damage and healing. probably with +spell healing too but I was playing as ret so I never tried it. One of the most overlooked items because its ability is passive and doesn't show up in the logs. I loved that sword. Saved my !@# so many times._
https://us.battle.net/forums/en/wow/topic/20760205815?page=8#post-145
